xbmc/network/CMakeLists.txt

```diff
@@ -48,3 +48,16 @@ core_add_library(network)
 if(BLUETOOTH_FOUND)
   target_compile_definitions(${CORE_LIBRARY} PRIVATE -DHAVE_LIBBLUETOOTH=1)
 endif()
+
+if(ENABLE_UPNP)
+  target_include_directories(${CORE_LIBRARY} PRIVATE ${CMAKE_SOURCE_DIR}
+                                             lib/libUPnP
```

Comment:
to work correctly this must be `${CMAKE_SOURCE_DIR}/lib/libUPnP`
same is true for the following lines